invenio_app_ils/documents/loaders/jsonschemas/document.py

```diff
@@ -197,6 +197,7 @@ class Meta:
     authors = fields.List(fields.Nested(AuthorSchema), required=True)
     conference_info = fields.Nested(ConferenceInfoSchema)
     copyrights = fields.List(fields.Nested(CopyrightsSchema))
+    cover_metadata = fields.Dict()
```

Comment:
I am thinking now that if you go to the editor, and update a field, e.g. the title, this will be empty and override any previous value... to be tested!
I think you should "preserve" the field, like if it was a system field, as done for the `created_by` and `updated_by`.
Here a small refactoring of the existing `@pre_load set_changed_by`:
```
    @pre_load
    def preload_fields(self, data, **kwargs):
        """Automatically inject system fields."""
        prev_record = self.context.get("record")
        data.update(set_changed_by(data, prev_record)))
        data.update(preserve_cover_metadata(data, prev_record))
        return data
```

Be aware that you have to do the same in the `series` loader.